### ISSUE TITLE
Incorporate Lighthouse audits into CI

### ISSUE THREAD

**Issue by twilco** [twilco/cratify#11]
To help ensure we do not regress in terms of accessibility, we should make Google Lighthouse audits a part of the CI process.  Maybe we should fail builds if we fall below some threshold?
